core/renderers/measurables/base.js

```diff
@@ -63,6 +63,14 @@ Blockly.blockRendering.Measurable.prototype.connectionShape =
 Blockly.blockRendering.Measurable.prototype.notchShape =
     Blockly.blockRendering.constants.NOTCH;
 
+/**
+ * The offset from the left side of a block or the inside of a statement input
+ * to the left side of the connection notch.
+ * @type {number}
+ */
+Blockly.blockRendering.Measurable.prototype.notchOffset =
```

Comment:
Can we define this on just the Measurables that will need it?  That is, previous and next connections, and statement inputs.

Reply:
I think notchOffset should be defined on all measurables that have a notchShape defined. So, I'd like to keep it there, but I wouldn't be against moving it along with notchShape if that refactor happens.

Reply:
+1